cypress/e2e/pages/articles/testsForCanonicalOnly.js

```diff
@@ -146,34 +141,26 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           }
         });
       });
+      if (service === 'pidgin') {
```

Comment:
Does this need to be a pidgin only test? Since we have the data now, we can use it to check if there is media on the page